documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -12,7 +12,7 @@
 
 toc::[]
 
-{prodname}'s Vitess connector captures row-level changes in the shards of a Vitess link:https://vitess.io/docs/concepts/keyspace/[keyspace]. Vitess versions 7 and 8 are supported.
+{prodname}'s Vitess connector captures row-level changes in the shards of a Vitess link:https://vitess.io/docs/concepts/keyspace/[keyspace]. Vitess version 8 is supported.
```

Comment:
I assume by this change this implies that Vitess 7 is no longer supported?  I ask just so we can make the appropriate adjustments at https://debezium.io/releases/.

Reply:
Hi @Naros, thanks for letting me know about https://debezium.io/releases/

Correct. We currently only focus on supporting the latest Vitess GA version, which is 8 at the moment. All integration tests are tested against Vitess 8. The reason being that the Vitess API is still changing rapidly. At some point, when Vitess API is becoming more stable, we can start considering supporting multiple Vitess versions.

Reply:
Makes sense. In particular as this connector still is incubating at this point; once it has matured, we'll need to consider compatibility for existing users, too.

Reply:
I definitely agree :)